tools/build/scripts/utils/addSourceDir.js

```diff
@@ -0,0 +1,22 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+
+const FS = require('fs-extra');
+const rootPackageJson = require('../../../../package.json');
```

Comment:
what root package.json is this always looking for?

Reply:
https://github.com/iTwin/itwinjs-core/blob/master/package.json, I then use the name "itwinjs-core".

Reply:
okay, not all of the packages in the repo are at that depth so not sure we can always use it. Although, I don't think those other packages are currently included in the docs but they probably should be...

Reply:
That's okay. That relative path is from this util script. For example, analytical-backend still gets a value of `domains\analytical\backend\src`.

Reply:
Ah okay, sounds good.